src/contrib/persistence/Akka.Persistence.Sql.Common/Journal/BatchingSqlJournal.cs

```diff
@@ -507,7 +515,7 @@ public RequestChunk(int chunkId, IJournalRequest[] requests)
         private readonly HashSet<IActorRef> _allIdsSubscribers;
         private readonly HashSet<string> _allPersistenceIds;
 
-        private readonly Func<Type, Serializer> _getSerializer;
+        private readonly Func<Type,string, Serializer> _getSerializer;
```

Comment:
@Horusiath why did you used this pattern? Why not simply call `Context.System.Serialization.FindSerializerFor` When where you need it? Why store a reference in a func ?